src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     * To be inherited by Email, Name, Address, and Phone - gives printable string representation of object
```

Comment:
The Javadoc for any method is supposed to be same whether it's before the function definition or declaration